bin/pasta_show_maintainers.py

```diff
@@ -0,0 +1,165 @@
+            counts_by_entry[subsystem] = np.add(counts_by_entry.get(subsystem, [0, 0]), file_lengths[filename])
+    return OrderedDict(sorted(counts_by_entry.items(), key=lambda x: x[1][1], reverse=True)), subsystem_dirs
+
+def get_maintainer_counts(all_maintainers, filenames = all_filenames):
+    maintainers_subs = dict()
+    counts_by_maintainer = dict()
+    for filename in filenames:
+        subsystems = all_maintainers.get_subsystems_by_file(filename)
+        for subsystem in subsystems:
+            maintainers = all_maintainers.get_maintainers(subsystem)
+            for maintainer in flatten(maintainers):
+                if not (type(maintainer) == str or maintainer[0] == '' or maintainer[1] == ''):
+                    maintainers_subs[maintainer[0]+' <'+maintainer[1]+'>'] = maintainers_subs.get(maintainer[0]+' <'+maintainer[1]+'>', set())
+                    maintainers_subs[maintainer[0]+' <'+maintainer[1]+'>'].add(subsystem)
+                    counts_by_maintainer[maintainer[0]+' <'+maintainer[1]+'>'] = np.add(counts_by_maintainer.get(maintainer[0]+' <'+maintainer[1]+'>', [0, 0]), file_lengths[filename])
+            
+    return OrderedDict(sorted(counts_by_maintainer.items(), key=lambda x: x[1][1], reverse=True)), maintainers_subs
+
+def get_small(title, counts):
+    results = list()
+    results.append([title, 'LoC'])
+    for entry in counts:
+        results.append([entry, counts[entry][0]])
+    return results
+
+def get_maintainers(config, sub, argv):
+    
+    parser = ArgumentParser(description='d')
+
+    parser.add_argument('maintainers', nargs=2, type=bool)
+    parser.add_argument('query', type=str, help='Group by subsystems or maintainers')
+    parser.add_argument('--smallstat', action='store_true', help='Simple view')
+    parser.add_argument('--outfile', type=str, help='Output to a csv file')
+    parser.add_argument('--file', type=str, help='Select MAINTAINERS file')
+    parser.add_argument('--filter', type=str, help='Filter by file list')
+
+    args = parser.parse_args()
+
+    if args.file!= None:
+        all_maintainers_text = file_to_string(args.file)
+    else:
+        all_maintainers_text = file_to_string('./resources/linux/repo/MAINTAINERS')
+    
+    if args.filter!= None:
+        filenames = file_to_string(args.filter).splitlines()
+
+    all_maintainers = LinuxMaintainers(all_maintainers_text)
+
+    if args.query == 'entries':
+        title = 'Entry'
+        counts, dirs= get_entry_counts(all_maintainers)
+        if args.filter != None:
+            counts_filt, dirs_filt = get_entry_counts(all_maintainers, filenames)
+    elif args.query == 'maintainers':
+        title = 'Maintainer'
+        counts, maintainers_subs = get_maintainer_counts(all_maintainers)
+        if args.filter != None:
+            counts_filt, _ = get_maintainer_counts(all_maintainers, filenames)
+    else:
+        return -1
```

Comment:
Probably here you should warn about wrong use and indicate how to get some help.